Docs/source/pages/get-started.md

```diff
@@ -29,6 +29,11 @@ To instrument your Unity application with AltTester Unity SDK you first need to
         1. Download from `AltTester website - link <https://alttester.com/app/uploads/AltTester/sdks/AltTesterPro.unitypackage>`_.
         2. Import it by drag and drop inside your Unity project.
 
+    .. tab:: UnityPackage from OpenUPM website
+
+        1. Go to `OpenUPM website - link <https://openupm.com/packages/com.alttester.sdk/>`_.
```

Comment:
Personally I would just leave `OpenUPM` without `website-link`. I see it is written this way in the other tab as well.